.travis.yml

```diff
@@ -24,5 +24,5 @@ script:
 - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && mvn -P travis clean install deploy || mvn -P travis clean install'
 env:
   global:
-  - secure: DHhZau8rR7tbHByl4I1A9ESVq+GjmyshEJzlfn573zlS5bTz94X/Xwn/8s7L4ZkT9HqTG/D/Hm9a7RaRJ6OqyFlTjw4q62JiKwHCDQp4GsLQ57JXhe5E2UWj8h8tObyv+mbeiWt/x0froG4xKkNaTBoPTcqbdVULF7xprbFRIwvy9E6IRlZT5zF+qNMpsxnmDJu9a+g1U6pwhXwn4+CIGi6G2dEkX6/DNPe1xKKA7ZmZ9wlR2o97PREZkXz/emuKy4V1RIMt4F28e2UHtyFbokjMnklmwQxvBF2e95vu2XGVWvjQoWg+1vVSyx09P/XiPeuYjZwJq9i4FuRAmo831p2KAK9NL+e/nR+TsqHWNE4wYbOYTxvYkcrCyyXm9sqUKEpKdqXmKTGjuL48ba4kVNADQvN4vapY5MUAh8hpAnI45fW2SqEj8xl5vBdMX1eE9jrR1JugWPZ7TGYMo/xsrztF8JpqEqV/oLUxad3F2idezZ9Ua26Huh2V0sUnhdhWyUoHpZsFYDTRfKTsQbcKjJtY49y+Ghu8yxsremJYdC94GKDKYsmPED5CQmPvHyG4C0NLOToZ8xbMKfIJXco6qZ0VzIvvlWEpUQ4kEP0RekG1clN7Ll32fOCI7rqhOYO94JvhJuzCR3nEgY8pEM16Ji4Tz/Q7+CQrIDSo7L4brzQ=
-  - secure: mHezyDR2h4MXyxj2IjYgezSrLkgIa3zgsmWlsHV28O2nVxIX1tbSWJ6jzYvNAtdonN8GHe9TCtz60SQCrTfZZ052jzAu3XrfR4VojL9iTnteLMfrzCzvSpmU/dbrlmgz6bhZmVbbZwH6CvutYaViaroXpKDHWbzWL/3eoWhnFIS+A8uXdyzShMFAZlU3zAxPVsQiC2pSxhQ99sFNdqnro22aCOoDEuaV6hUT6XXVqgPBgPVjdxIIT9/OwN1pzcOTA9UNoSFdRHCif/k+ICoKv59c9MfM3SAR1qeUydLeK/ITDS/FjE+z0+8rMSzHqTmnej6+x2/ZEYfKcmoRdxtGqn4s/TEJx1qXRnEAcnK9zfKhhJGU65/rV6BCEHad1L51I8YRxazeBWaxGm27ervA7pVkjOSh+sAyPSHeu7kqoCMwbvzN2BpCnzwfZQOm+/4DQrEzw5gqXVFDUnhRvekutlqsprZLe1oAcnALlMF20N++Gj2riq+7TIaa0v7C3sRpcr9fXPlNP30SzDO5lmV3hbWuMtOx3tSKoGfR6v9KCMZwLcIYxfS3m+/QHI56lMaV3LXI8nMd/5wHe6ZAIKIfgAe3wfIB+7C5N9ZUioqhigeAiqUElJIwx2TuRtwC9XB5gVgXGpCtBoXlKdVeu28nvylTu77R8Mjlf4TNFwSwe5s=
+  - secure: qhW/upWDYH/QjOvSLE6rp9wfnGQwgBIt3IXLLRRREbtMJiBdn9eNpzgsTc+z9yljDd1de5gtfM7D1yCk1eaXqXF9E+HzuE45YFDfE5ifKwNOU9I+8b6KC9QMMUeI7mxK1mCRtlTY36G08tS5djFkKBx/m7PbTtd9PAyfqUt261OzxYx9eTW8qLI+W47p4w6t6d9kjxXuRarDN6ESTNAcWvW7b7SZ1qiH3yKedBIL+FMJ8ms4U9RhHKosPv8bLUUgvtcTbZLJ7nqqM8+9zeHQO1Otp4D+j26FFvxpNq0LGBPLpBc3M/kVqNxc52G7kFYUmb3Y0I6uW+aj50tvI6Qr5JU9g920NiLNJ84VbdJKVJsFdC8flz9W9G0y5OLCMnobJTCw7woFYLZ/mFyopMl/zrc+YI0Dd3MRw5ePlkPMrQgj6vDsqdHOgrayD5b3AUo6p59wtNYuVfcgmOfxy+MwZlvcO6S0OMfav6HuFW40yc5LrbVvnvNobm0eVcTY9JNAkCl2w+9CJR5gKw4s24+V4HbZZdTjpm9CyEHjb94gA4/0/SB9B4aukC5mdJqRotPS1N9ZEDzlJ6dMtyrVcIizznIayqScNvQqSKQXmfPxhcIo8l/nfUoKz0xgp9jzxlq5ywGdCyn4ihqHOra0vz0ssoYMWY6mxvtUHHpGW4HMwsM=
```

Comment:
These are different to the secrets I fixed on the `master` branch but did not also fix on the `develop` branch in https://github.com/AtlasOfLivingAustralia/ala-sensitive-data-service/commit/59dfc6636d1f54d26c5f492f97424151d652c501 

It would be useful to merge master into develop as there is also the issue of `oraclejdk8` which is switching to `openjdk8`

Reply:
Merged master with new secrets.

Reply:
Done